GenderPayGap.WebUI/Views/Submit/CheckData.cshtml

```diff
@@ -76,27 +76,37 @@
         <thead>
             <tr>
                 <th colspan="2">
-                    <span>Hourly rate</span>

```

Comment:
**Note for whole file:**
The fields in the file have been re-ordered - but it makes the diff look like I've changed everything!